Tests/MongoSwiftTests/ConnectionStringTests.swift

```diff
@@ -104,10 +104,6 @@ let shouldWarnButLibmongocErrors: [String: [String]] = [
         "Non-numeric wTimeoutMS causes a warning",
         "Too low wTimeoutMS causes a warning",
         "Invalid journal causes a warning"
-    ],
-    "valid-warnings.json": [
-        "Empty integer option values are ignored",
-        "Empty boolean option value are ignored"
```

Comment:
libmongoc stopped erroring in these cases. I would prefer if we continued to error too, but I'm not sure if it's possible short of us writing our own URI parser.

Reply:
ah, so I meant that I think we should:
1) accept the libmongoc changes to comply with the spec as-is rather than trying to work around them, but also
2) plan that when we do write our own URI parser, we start erroring here again (I agree with you that I'd prefer to error)

I think this is consistent with how we handled https://jira.mongodb.org/browse/SWIFT-951; in general (in Swift) we have opted to error on invalid values of known options

Reply:
Af okay, makes sense. Filed SWIFT-1009.